tracer/build/_build/Build.Steps.cs

```diff
@@ -1189,6 +1242,61 @@ async Task DownloadWafVersion(string libddwafVersion = null, string uncompressFo
             }
         });
 
+    Target RunDebuggerIntegrationTests => _ => _
+        .Unlisted()
+        .After(BuildTracerHome)
+        .After(BuildDebuggerIntegrationTests)
+        .Requires(() => Framework)
+        .Triggers(PrintSnapshotsDiff)
+        .Executes(() =>
+        {
+            EnsureExistingDirectory(TestLogsDirectory);
+            EnsureResultsDirectory(DebuggerIntegrationTests);
+
+            try
+            {
+                DotNetTest(config => config
+                    .SetDotnetPath(TargetPlatform)
+                    .SetConfiguration(BuildConfiguration)
+                    .SetTargetPlatform(TargetPlatform)
+                    .SetFramework(Framework)
+                    .EnableCrashDumps()
+                    .EnableNoRestore()
+                    .EnableNoBuild()
+                    .SetFilter(GetTestFilter())
+                    .SetProcessEnvironmentVariable("MonitoringHomeDirectory", MonitoringHomeDirectory)
+                    .SetLogsDirectory(TestLogsDirectory)
+                    .When(!string.IsNullOrEmpty(Filter), c => c.SetFilter(Filter))
```

Comment:
You're calling `SetFilter()` twice here
```suggestion
```

Reply:
Oh, right. 
Removed 246e272f4eb30b9aacf8302d55794f1774caa194